### ISSUE TITLE
json_decode(): integer overflow detected in ConfigFileStorage->decode() (line 982 ...core/includes/config.inc).

### ISSUE THREAD

**Issue by clickbox** [backdrop-contrib/backup_migrate#14]
Maybe I didn't follow the installation instructions correctly but I get a ton of these errors after enabling this module on my apache/ubuntu server.

`json_decode(): integer overflow detected in ConfigFileStorage->decode() (line 982 of /var/www/live_ezenwaisiogu/core/includes/config.inc).`

**Comment by LeeteqXV** [comment]
I just did a fresh install of BD 1.4.2 (PHP version 5.6.20, 192 MB RAM), and also getting lots of those entries in the log right after enabling (only) Backup and Migrate module, which is the only extra module that has been enabled on this new site.

**Some background info for troubleshooting:**
Timeout set to 120 (2 minutes) in settings.php, and I installed (downloaded) 20 modules simultaneously directly towards the BD server (no manual uploads). At that stage I received a timeout, and only the Captcha module and OAuth modules were not fetched. Tried again, no luck with OAuth, but Captcha module got through after 3 retries. No more timeouts. No errors of the reported type on the modules page during or after installation, they only started right after I enabled Backup and Migrate from the modules page after I had fetched those 19 modules.

I am not seeing those errors on every page, these are the location urls from the log entries:
admin/modules/list/confirm
admin/config/system/backup_migrate
admin/config/system/backup_migrate/export/advanced

Looks like I get roughly 60 identical entries on each page load where that error occurs.
